docs/src/main/sphinx/connector/redshift.rst

```diff
@@ -109,6 +109,145 @@ that catalog name instead of ``redshift`` in the above examples.
 Type mapping
 ------------
 
+Because Trino and Redshift each support types that the other does not, this
```

Comment:
Unfortunately, Redshift connector isn't maintained due to lack of CI environment. These type mapping aren't tested at all. I hesitate to merge this PR in this situation.